src/log.ts

```diff
@@ -60,35 +69,6 @@ export interface ApiResponseCallback {
 }
 export type DeleteCallback = ApiResponseCallback;
 
-export type MonitoredResource = google.api.IMonitoredResource;
-export interface WriteOptions {
-  dryRun?: boolean;
-  gaxOptions?: CallOptions;
-  labels?: {[index: string]: string};
-  partialSuccess?: boolean;
-  resource?: MonitoredResource;
-}
-
-export enum Severity {
-  emergency,
-  alert,
-  critical,
-  error,
-  warning,
-  notice,
-  info,
-  debug,
-}
```

Comment:
is it safe to remove this? Will it affect current users?

Reply:
I refactored it to log-common to be shared between Log and LogSync classes